src/CustomInput.js

```diff
@@ -7,7 +7,7 @@ const propTypes = {
   className: PropTypes.string,
   id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
   type: PropTypes.string.isRequired,
-  label: PropTypes.string,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
probably just want `PropTypes.node` which is defined as:Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.